sydent/sydent.py

```diff
@@ -482,7 +482,7 @@ def get_branded_template(self, brand, template_name, deprecated_template_name):
         if os.path.exists(
             os.path.join(root_template_path, brand, template_name + ".j2")
         ):
-            return os.path.join(root_template_path, brand, template_name + ".j2")
+            return os.path.join(brand, template_name + ".j2")
```

Comment:
confused about this. I suspect it was right before, but did you have a reason to change it?

Reply:
This change was as a result of this, https://github.com/matrix-org/sydent/pull/376#discussion_r694045962
The result was that I changed how jinja templates were handled by `get_branded_template` as discussed in the comment, thus this change was necessary

Reply:
Aha, thanks

Reply:
See this thread: https://github.com/matrix-org/sydent/pull/376#discussion_r694045962

Reply:
I should note that this is kind of hacky, but it seemed like the clearest thing to do?